elastica/joint.py

```diff
@@ -209,9 +210,14 @@ class FixedJoint(FreeJoint):
             Damping coefficient of the joint.
         kt: float
             Rotational stiffness coefficient of the joint.
+        nut: float
+            Rotational damping coefficient of the joint.
+        static_rotation: np.array
+            Static 3x3 rotation matrix from system one to system two.
+            Instead of aligning the directors of both systems directly, a desired rotational offset C_12* is enforced.
```

Comment:
```suggestion
            Rest 3x3 rotation matrix from system one to system two at the connected elements.
            Instead of aligning the directors of both systems directly, a desired rest rotational matrix labeled C_12 is enforced.
```